api/v1alpha1/ingressnodefirewall_types.go

```diff
@@ -154,8 +154,14 @@ type IngressNodeFirewallSpec struct {
 }
 
 // IngressNodeFirewallStatus defines the observed state of IngressNodeFirewall.
-type IngressNodeFirewallStatus struct {
```

Comment:
I'd prefer to keep `IngressNodeFirewallStatus` reserved for this struct in case that we want to add a real obj.status field here. Can we keep the empty struct and rename the type ... string in line 157?